src/test/groovy/net/rubygrapefruit/platform/file/AbstractFileEventsTest.groovy

```diff
@@ -551,7 +604,7 @@ abstract class AbstractFileEventsTest extends Specification {
     }
 
     @Unroll
-    @IgnoreIf({ Platform.current().windows })
+    @IgnoreIf({ Platform.current().windows || Platform.current().linux })
```

Comment:
Is this something we expect to fix for Linux and Windows? Can you add a comment?

Reply:
Added.